Platform/CommonBoardPkg/Library/BoardSupportLib/BoardSupportLib.c

```diff
@@ -160,19 +160,18 @@ PlatformNameInit (
 
 
 /**
-  Load the configuration data blob from SPI flash into destination buffer.
-  It supports the sources:  BIOS for external Cfgdata.
-
-  @param[in]    Dst        Destination address to load configuration data blob.
-  @param[in]    Src        Source address to load configuration data blob.
-  @param[in]    Len        The destination address buffer size.
-
-  @retval  EFI_SUCCESS             Configuration data blob was loaded successfully.
-  @retval  EFI_NOT_FOUND           Configuration data blob cannot be found.
-  @retval  EFI_OUT_OF_RESOURCES    Destination buffer is too small to hold the
-                                   configuration data blob.
-  @retval  Others                  Failed to load configuration data blob.
-
+  Set framebuffer range as writecombining for performance.
+
+  @param[in]    FrameBufferBase   Framebuffer base address.
+                                  if 0, it will use framebuffer HOB to get the base.
+  @param[in]    FrameBufferSize   Framebuffer size.
+                                  if 0, it will use framebuffer HOB to get the size.
+                                  if MAX_UINT32, it will parse the PCI bar to get the size.
+
+  @retval  EFI_SUCCESS            The framebuffer cache was enabled successfully.
+           EFI_NOT_FOUND          Failed to find the required GFX controller.
+           EFI_UNSUPPORTED        The base and size cannot be supported.
+           EFI_OUT_OF_RESOURCES   No enough MTRR to use.
 **/

```

Comment:
I guess this change is a typo. This comment change doesn't match with the function.

Reply:
ahh, something was copied incorrectly.  will update it.